templates/types/streaming/nextjs/app/components/ui/chat/index.ts

```diff
@@ -13,6 +13,7 @@ export enum MessageAnnotationType {
   EVENTS = "events",
   TOOLS = "tools",
   SUGGESTED_QUESTIONS = "suggested_questions",
+  AGENT_EVENTS = "agent_events",
```

Comment:
let's just call this `agent`
```suggestion
  AGENT_EVENTS = "agent",
```